arcgis-ios-sdk-samples/Maps/Download preplanned map/README.md

```diff
@@ -0,0 +1,47 @@
+# Download preplanned map
+Take a map offline using a preplanned map area.
+
+![](image1.png)
```

Comment:
You might consider adding an additional screenshot showing the popup or just showing the popup in this screenshot.

Reply:
Considered & updated in [76573d2b](https://github.com/Esri/arcgis-runtime-samples-ios/pull/758/commits/76573d2b7c82d46d22188e22aa137d5bc6c03f3b).